### PR TITLE
Operating System Refs new indicator field

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/33656
bloked by: https://github.com/demisto/etc/issues/33806
## Description
Add a new indicator field `Operating System Refs`.

### CODE REVIEW

**Comment by ChanochShayner** [comment]
@ytauman Do we want it to be `shortText` type or `tagsSelect` type?

File: Packs/CommonTypes/IndicatorFields/indicatorfield-operating_system_refs.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "associatedToAll": true,
```
Comment (bakatzir) [inline Packs/CommonTypes/IndicatorFields/indicatorfield-operating_system_refs.json:20]: isn't is supposed to be associated to a specific IOC?

Comment (ChanochShayner) [reply Packs/CommonTypes/IndicatorFields/indicatorfield-operating_system_refs.json]: added

Comment (bakatzir) [reply Packs/CommonTypes/IndicatorFields/indicatorfield-operating_system_refs.json]: please check that, I think it should be the details filed, which is, e.g `STIX Malware`

Comment (ChanochShayner) [reply Packs/CommonTypes/IndicatorFields/indicatorfield-operating_system_refs.json]: This is a case where we do not have a complete list of options, so we thought (me and @ytauman) we should use `tags` (as in stix aliases).

**Review by bakatzir — APPROVED** [review]

**Comment by ChanochShayner** [comment]
I also added to this PR a transfer of the `MITER ID` indicator field from the `MITREAttackv1` pack to `CommonTypes` and added a new type to it.
@bakatzir @ytauman @yuvalbenshalom FYI

**Review by bakatzir — COMMENTED** [review]
That's ok, but let's make sure to do 2 things:
1. bump the mitre and commontypes packs, and update a mandatory dep in between mitre pack to commontypes.
2. add an rn note in the mitre pack about the braking change, and add that without installing the commontypes pack uptodate, it will not work.

File: Packs/FeedMitreAttack/ReleaseNotes/1_1_13.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Indicator Fields
+**MITRE ID**
+- Moved the indicator field to the CommonTypes pack. Update the CommonTypes pack to version x.x.x and onwards to continue working seamlessly with this pack.
```
Comment (ChanochShayner) [inline Packs/FeedMitreAttack/ReleaseNotes/1_1_13.md:4]:
```suggestion
- Moved the indicator field to the CommonTypes pack. Update the CommonTypes pack to version #TODO and onwards to continue working seamlessly with this pack.
```

File: Packs/FeedMitreAttack/pack_metadata.json [hunk 3]
```diff
@@ -16,5 +16,11 @@
...
         "Feed",
         "ATT&CK"
-    ]
+    ],
+    "dependencies": {
+        "GetLicenseID": {
+            "mandatory": true,
+            "name": "Get License ID"
+        }
+    }
```
Comment (bakatzir) [inline Packs/FeedMitreAttack/pack_metadata.json:25]: why is this needed?

Comment (ChanochShayner) [reply Packs/FeedMitreAttack/pack_metadata.json]: fixed

**Review by bakatzir — APPROVED** [review]

**Review by gal-berger — APPROVED** [review]

**Review by yuvalbenshalom — APPROVED** [review]

File: Packs/CommonTypes/ReleaseNotes/2_9_8.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+
+#### Indicator Fields
+New indicator fields:
+- **Operating System Refs**
+- **MITRE ID**
+
+- Associated the **Kill Chain Phases** field with the following *Attack Pattern* and *Tool* indicator types.
```
Comment (amshamah419) [inline Packs/CommonTypes/ReleaseNotes/2_9_8.md:7]:
```suggestion
- Associated the **Kill Chain Phases** field with the *Attack Pattern* and *Tool* indicator types.
```

File: Packs/FeedMitreAttack/ReleaseNotes/1_1_13.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+#### Indicator Fields
+**MITRE ID**
+- Moved the indicator field to the CommonTypes pack. Update the CommonTypes pack to version 2.9.8 and onwards to continue working seamlessly with this pack.
```
Comment (amshamah419) [inline Packs/FeedMitreAttack/ReleaseNotes/1_1_13.md:4]:
```suggestion
- Moved the indicator field to the CommonTypes pack. Update the CommonTypes pack to minimum version 2.9.8 to continue working with this pack.
```
